packages/design-preview/src/components/style.scss

```diff
@@ -262,6 +262,27 @@ $break-design-preview: 1024px;
 		margin-bottom: 32px;
 	}
 
+	.design-preview__site-preview-image {
```

Comment:
where do we use this class name?

Reply:
Good catch, we don't need this anymore 👍